hip_data_tools/etl/google_sheet_to_athena.py

```diff
@@ -27,11 +28,19 @@ class GoogleSheetsToAthenaSettings:
     Google sheets to Athena ETL settings
     Args:
         workbook_name: the name of the workbook (eg: Tradie Acquisition Targets)
+            If there are multiple workbooks with the same name, the workbook which has most recently
+            shared with the google service account is used
         sheet_name: name of the google sheet (eg: sheet1)
         row_range: range of rows (eg: '2:5')
         table_name: name of the athena table (eg: 'sheet_table')
-        fields: list of sheet field names and types. Field names cannot contain hyphens('-')
+        fields: list of sheet field names and types. Field names cannot contain hyphens('-'), spaces
+            and special characters
             (eg: ['name:string','age:number','is_member:boolean'])
+            If this is None, the field names and types from google sheet are used automatically
+        field_names_row_number: row number of the field names (eg: 4)
```

Comment:
assumes the data starts at first column and there is no gaps (columns without data)